docs/docs/pii-management.mdx

```diff
@@ -0,0 +1,169 @@
+---
+id: pii-management
+sidebar_label: PII Management
+title: PII Management
+hide_table_of_contents: false
+---
+import RasaProLabel from "@theme/RasaProLabel";
+
+import RasaProBanner from "@theme/RasaProBanner";
+
+<RasaProLabel />
+
+<RasaProBanner />
+
+:::info New in 3.6
+
+You can now anonymize personally identifiable information (PII) data in your logs and events streamed via Kafka event
+broker only. Continue reading to learn how to enable the anonymization of PII data.
+
+:::
+
+Management of sensitive customer data collected by assistants is a critical requirement for complying with regulations
+and managing the data securely while making it available for analysis.
+Rasa Pro 3.6 introduces the ability to anonymize PII data in your logs and events streamed via the Kafka event broker.
+Note that the tracker store continues to store conversation data which is not anonymized because the tracker store data
```

Comment:
Can we make it more clear here why we are not anonimizing the tracker store in this version? 
Eg.: we want to retain a source of truth for pristine data for non-repudiation purposes. And probably also prediction..? (not sure)

Reply:
Addressed 👍🏻 
https://github.com/RasaHQ/rasa/pull/12505/commits/f1b891cdcec2108822c99ebc3dc629c88d44c5b4